grc/utils/application_files.py

```diff
@@ -42,28 +47,37 @@ def create_or_download_attachments(self, reference_number, application, download
         return bytes, file_name
 
 
-    def create_or_download_pdf(self, reference_number, application, is_admin=True, download=False):
+    def create_or_download_pdf(self, reference_number, application, is_admin=True, attach_files=True, download=False):
         bytes = None
         file_name = ''
 
         try:
             file_name = reference_number + '.pdf' if is_admin else 'grc_' + str(application['email']).replace('@', '_').replace('.', '_') + '.pdf'
 
             data = None
-            if is_admin:
-                data = AwsS3Client().download_object(file_name)
+            if is_admin and not attach_files:
+                data = None if os.getenv('FLASK_ENV', '') == 'development' else AwsS3Client().download_object(file_name)
```

Comment:
Similarly, is there a reason we don't want this to run on our laptops?

Reply:
Same as above

Reply:
👍